include/acl.h

```diff
@@ -514,23 +514,17 @@ typedef enum {
 
 // Enum values here also need to match the SPIRV spec for device
```

Comment:
Since the enum was removed, I think you can also remove this part of the comment.